.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: php
+
+php:
+  - '7.1'
+
+jobs:
+  include:
+    - stage: test
+      script: make phpcs
```

Comment:
you could also use https://prettyci.com/ as a separate ci hook, for nicer output of cs issues

Reply:
cool, didn't know that. added it now it seems to work :)